### PR TITLE
Adding recommended AWS metrics tables to Readme files

### PR DESCRIPTION
**Pull request by alstory**
Added recommended AWS metrics for the time they get implemented. Services with recommended metrics are: ALB, API Gateway, DynamoDB, EBS, EC2, ECS, EKS, ElasticCache, ELS, Lambda, RDS, Redshift, Route53, SNS, SQS, AutoScaling, CloudFront, Kinesis Streams, OpsWorks.

### CODE REVIEW

File: aws-ec2/README.md [hunk 1]
```diff
@@ -72,6 +73,68 @@ For more information about the filters, see <a target="_blank" href="http://docs
...
 For more information about the metrics emitted by Amazon EC2, visit the service's homepage at <a target="_blank" href="https://aws.amazon.com/ec2/">https://aws.amazon.com/ec2/</a>.
 
+### RECOMMENDED Metrics
+
+The following are a subset of available metrics; these are recommended by Amazon for collection.
+
+
+_Standard EC2 Metrics_
+
+CPUUtilization
```
Comment (jadbSFx) [inline aws-ec2/README.md]: Many of these metrics do not list a recommended stat. I asked @asuresh4 about this and this implies that all 5 stats are collected. My recommendation would be to explicitly state that for each metric so there is no confusion (as it's entirely possible somebody would not read a comment at the top of the list or a general comment on all metrics).

Comment (jadbSFx) [inline aws-ec2/README.md]: This list is pretty plain... I would suggest we present these as a table

**Comment by mstumpfx** [comment]
@alstory  Above each `### Recommended Stats` you will need to add `<!--- METRICS --->` if you want this to render in the UI under the metrics tab. Otherwise, this will not show up in app. That format is also new for Ben's re-work of documentation, so we'll have to wait for his stuff to merge first as well.

File: aws-ec2/README.md (hunk 1 above)
Comment (alstory) [reply aws-ec2/README.md]: Done.

Comment (alstory) [reply aws-ec2/README.md]: Done.

**Pull request merged**